tests/System.Text.CaseFolding.Tests/Unicode/TestUnicodeSimpleCaseFoldingFoldSpan.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.CaseFolding;
+using System.Text.CaseFolding.Tests;
+using Xunit;
+
+namespace System.Text.CaseFolding.Tests
+{
+    public class FoldStringAndSpanTests
+    {
+        [Fact]
+        public static void Fold_String_Span_Roundtrips()
+        {
+            string s = "Turkish I \u0131s TROUBL\u0130NG!";
+            ReadOnlySpan<char> span = s.AsSpan();
+            Span<char> span2 = stackalloc char[s.Length];
```

Comment:
No need to use stackalloc/etc. in test code:
```C#
Span<char> spanCopy = span.ToArray();
```

Reply:
The pattern was copy-pasted from CoreFX. Perhaps there is a trick in the test. I did not dare to change anything.